homework#1-2

Dikinz opened the pull request:
Выполнил домашнее задание 1-2
- страничка выполнена по БЭМ
- добавлены картинки и их размеры
- установлены стили
- сделана разметка header
- сделана адаптация - хидера  под @media - 768

Dikinz committed:
homework#1-2

- страничка выполнена по БЭМ
- добавлены картинки и их размеры
- установлены стили
- сделана разметка header
- сделана адаптация под @media

WebnautsAcademy commented on source/index.html:
```diff
@@ -1,11 +1,309 @@
+					<li class="list__item">
+						Увеличение продаж
+					</li>
+					<p class="list__text">
```
p - не может быть ребенком ul. Надо его поместить в li. И содержимое p в данном случае - это выпадающий текст, как во многих блоках с FAQ делают. Поэтому, тот текст, что Вы в li поместили - заверните header или div. Туда же ещё кнопку поместите, которую Вы потеряли. В макете есть. А вот под этой оберткой - разместите этот p.

WebnautsAcademy commented on source/index.html:
```diff
@@ -1,11 +1,309 @@
+				</ul>
+			</div>
+		</section>
+		<section class="block4">
```
Все секции лучше называть говорящими классами. Например: about, reviews и т.д

WebnautsAcademy commented on source/index.html:
```diff
@@ -1,11 +1,309 @@
+							fill="white" />
+					</svg>
+				</div>
+				<button class="block4__btn1">
```
Кнопкам тоже классы и модификаторы лучше задавать говорящими названиями. Тут next и previous к примеру, а не 1 и 2

WebnautsAcademy commented on source/index.html:
```diff
@@ -1,11 +1,309 @@
+						</a>
+					</li>
+					<li class="list2__item item">
+						<a href="#" class="item__privacy">
```
Это не ссылка - это просто копирайт-текст

WebnautsAcademy commented on source/index.html:
```diff
@@ -1,11 +1,309 @@
+								fill="#4F4F4F" />
+						</svg>
+					</li>
+					<li class="list1__item item">
```
item для блока - такое себе название... Итемов у вам миллион на проекте может быть. Задавайте говорящие названия классам! Вы же не конструктор... Я ещё понимаю, если бы вы всё js кодом строили... Но руками и такое - не хорошо.

WebnautsAcademy commented on source/less/variables.less:
```diff
@@ -0,0 +1,12 @@
+@bebasNeueBold: 'BebasNeueBold', sans-serif;
```
Переменная для одного семейства шрифтов - одна. Жирность потом задается через font-weight: bold; и т.д.

WebnautsAcademy commented on source/less/style.less:
```diff
@@ -0,0 +1,10 @@
+@import 'mixins.less';
```
Всё, что не в папке blocks лучше завернуть в папку global. На первом уровне остаться должен только style.less

WebnautsAcademy reviewed (commented):
Необходимо изменить всем блокам, где есть цифры, классы на говорящие.

Потом создать под них less блоки.

И после этого - подключить эти блоки в style.less

Так же поправьте прочие замечания и рекомендации, что содержатся в комментариях.

ДЗ пока не принято.

Исправьте всё в новых коммитах. Потом повторно пройдусь по внесённым изменениям.

Удачи )

Dikinz committed:
update 17.07

- обновил названия секций на говорящие
- создал папку global  и поместил всё содержимое кроме style.less внутрь.

WebnautsAcademy commented on source/img/sprite.svg:
```diff
@@ -0,0 +1,5 @@
+<svg > 
+<symbolxmlns="http://www.w3.org/2000/svg" id='sunny'>
```
Тут пробел пропущен в открывающемся symbol

WebnautsAcademy reviewed (commented):
Теперь, вроде как - всё правильно.

Только если спрайт руками создавали - то пробел потеряли. 

ДЗ 1-2 - принимаю.

Подтяните из оригинального майн изменения в свой форк, а потом - в локальный репозиторий.
